### ISSUE TITLE
this can deploy in web?

### ISSUE THREAD

**Issue by johnfelipe** [PedroRavaglia/AI-Virtual-Painter#1]

**Comment by PedroRavaglia** [comment]
No, but in the [mediapipe](https://github.com/google/mediapipe) library that I used there is also a version in javascript of the [mediapipe hands](https://google.github.io/mediapipe/solutions/hands). So I think that you can make a new project using the same ideas that I used, but using the javascript version to deploy to the web.

**Comment by vnr235** [comment]
which version of openCV you have been used in this project.
cv2.imshow('MediaPipe Hands', img)
cv2.error: OpenCV(4.7.0) D:\a\opencv-python\opencv-python\opencv\modules\highgui\src\window.cpp:1272: error: (-2:Unspecified error) The function is not implemented. Rebuild the library with Windows, GTK+ 2.x or Cocoa support. If you are on Ubuntu or Debian, install libgtk2.0-dev and pkg-config, then re-run cmake or configure script in function 'cvShowImage'

this error is shown in my system.
